cli/lint_reporter.rs

```diff
@@ -0,0 +1,125 @@
+// Copyright 2018-2020 the Deno authors. All rights reserved. MIT license.
+
+//! This module defines an abstraction LintReporter, which is something that knows
+//! how to report observed lints (for example, pretty print or json).
+use crate::colors;
+use crate::fmt_errors;
+use crate::lint_serialize::SerializableLint;
+use deno_core::ErrBox;
+use deno_lint::diagnostic::LintDiagnostic;
+use serde::Serialize;
+
+pub enum LintReporterKind {
+  Console,
```

Comment:
**thought:** I realize now that this should probably be called `Pretty` 😅 What do you think?

Reply:
Agreed. At the moment all linting writes to the console. Also, this might be confusing if we ever want to add different output targets.

Reply:
Done! Renamed `ConsoleX` to `PrettyX`